source/types/ProxyRequestHandler.ts

```diff
@@ -0,0 +1,10 @@
+import { ClientRequest } from 'http';
+import { Request, Response } from '../interfaces';
+
+type ProxyRequestHandler = (
```

Comment:
We had to add this type since `http-proxy-middleware` doesn't export it :confused: